deps/wazuh_testing/wazuh_testing/tools/file.py

```diff
@@ -235,10 +235,33 @@ def delete_file(file_path):
 
 
 def delete_path_recursively(path):
+    '''Remove a directory recursively.
+
+    Args:
+        path (str): Directory path.
+    '''
     if os.path.exists(path):
         shutil.rmtree(path, onerror=on_write_error)
 
 
+def remove_files_in_folder(dir_path):
```

Comment:
Use the `file.delete_all_files_in_folder` function from `4.5` branch

Reply:
Done in [8a48b86](https://github.com/wazuh/wazuh-qa/pull/3837/commits/8a48b86915fca9d798ad4f96f2972b288fbf414f)